src/components/main/main.js

```diff
@@ -110,6 +111,10 @@ app.component('main', {
             delete res.publicKey;
           }
 
+          this.notify.about(
+            'deposite',
```

Comment:
Please correct the typo: deposite -> deposit. Please make sure it's fixed everywhere.